src/core/tests/runtime/interpreter/evaluates_map.cpp

```diff
@@ -2775,5 +2785,6 @@ runtime::interpreter::EvaluatorsMap& runtime::interpreter::get_evaluators_map()
 
 #undef NGRAPH_OP
     };
+    evaluatorsMap[snippets::op::Subgraph::get_type_info_static()] = evaluate_node<snippets::op::Subgraph>;
     return evaluatorsMap;
```

Comment:
why is it needed? Interpreter works on reference ngraph funciton

Reply:
We used interpreterFunction in some Snippets-specific [tests](https://github.com/IvanNovoselov/openvino/blob/bb8cc42588a02bad7b638bf39befbf9dd1a30229/inference-engine/tests/unit/cpu/generate_add.cpp#L54-L55). I agree that it's worth to look for a better solution, so I removed snippets from evaluatorsMap for now.